package com.example.UserDept.exceptions;

import java.io.Serial;

public class ResourceNotFoundException extends RuntimeException {
	public ResourceNotFoundException(Object id) {
		super("Recurso n√£o encontrado com id: " + id);
	}
}
